.github/workflows/release.yml

```diff
@@ -1,33 +1,38 @@
-name: release
+name: Release
+
 on:
   push:
     tags:
       - 'v*'
-      
-permissions:
-  contents: write # To be able to add artifacts to releases
```

Comment:
@imjasonh wasn't this needed?

Reply:
i just move that to the job and not to be global :)